Lib/sqlite3/__main__.py

```diff
@@ -41,41 +41,206 @@ def execute(c, sql, suppress_errors=True, theme=theme_no_color):
 
 class SqliteInteractiveConsole(InteractiveConsole):
     """A simple SQLite REPL."""
+    PS1 = "sqlite> "
+    PS2 = "    ... "
+    ruler = "="
+    doc_header = "Documented commands (type .help <command>):"
+    undoc_header = "Undocumented commands:"
 
     def __init__(self, connection, use_color=False):
         super().__init__()
         self._con = connection
         self._cur = connection.cursor()
         self._use_color = use_color
+        self._theme = get_theme(force_no_color=not use_color)
+
+        s = self._theme.syntax
+        sys.ps1 = f"{s.prompt}{self.PS1}{s.reset}"
+        sys.ps2 = f"{s.prompt}{self.PS2}{s.reset}"
+
+    def do_version(self, _):
+        """.version
+
+        Show version of the runtime SQLite library.
```

Comment:
```suggestion
        Print version of the runtime SQLite library.
```
This is the more common way of describing this. And also this does not need a new function.

Reply:
Just as explained at https://github.com/python/cpython/pull/133935#discussion_r2106114810, we do need a function here.